opentelemetry-jaeger/src/exporter/config/agent.rs

```diff
@@ -226,6 +232,24 @@ impl AgentPipeline {
         self
     }
 
+    /// Assign the batch span processor for the exporter pipeline.
```

Comment:
May be worth to mention if a simple span processor is used/installed then this config is ignored.